help_test.go

```diff
@@ -574,58 +574,6 @@ func TestShowCommandHelp_CommandAliases(t *testing.T) {
 	}
 }
 
-func TestHelpNameConsistency(t *testing.T) {
```

Comment:
@meatballhat Any reason why this was removed ?

Reply:
Oh gosh I totally don't remember why I did this 😭 I'm starting over!

Reply:
Ooh, I remember! But I'm still starting over 😂 I'll be introducing smaller PRs to do parts of what's discovered here.